pkg/installer/aromanifests.go

```diff
@@ -0,0 +1,114 @@
+package installer
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/asset/ignition"
+	"github.com/openshift/installer/pkg/asset/ignition/bootstrap"
+	"github.com/pkg/errors"
+
+	"github.com/openshift/installer-aro-wrapper/pkg/cluster/graph"
+)
+
+const (
+	aroManifestDir = "manifests"
+	rootPath       = "/opt/openshift"
+)
+
+// Custom ARO asset to add custom manifests to install graph in installer-wrapper similar to installer's manifests.Manifests
+type AROManifests struct {
+	FileList []*asset.File
+}
+
+// ARO File Fetcher to read manifests
+type aroFileFetcher struct {
+	directory string
+}
+
+var (
+	_ asset.WritableAsset = (*AROManifests)(nil)
+	_ asset.FileFetcher   = (*aroFileFetcher)(nil)
+)
+
+func (am *AROManifests) Name() string {
+	return "ARO Manifests"
+}
+
+func (am *AROManifests) Dependencies() []asset.Asset {
+	return []asset.Asset{}
+}
+
+func (am *AROManifests) Generate(dependencies asset.Parents) error {
+	return nil
+}
+
+func (am *AROManifests) Files() []*asset.File {
+	return am.FileList
+}
+
+func (am *AROManifests) Load(f asset.FileFetcher) (found bool, err error) {
+	yamlFileList, err := f.FetchByPattern(filepath.Join(aroManifestDir, "*.yaml"))
+	if err != nil {
+		return false, errors.Wrap(err, "failed to load *.yaml files")
+	}
+	ymlFileList, err := f.FetchByPattern(filepath.Join(aroManifestDir, "*.yml"))
+	if err != nil {
+		return false, errors.Wrap(err, "failed to load *.yml files")
+	}
```

Comment:
Kind of a nitpick, but do we need yml? Looking at https://github.com/Azure/ARO-RP/pull/3802/files, it looks like our manifests will always be yaml.